tests/unit/tx-pool.test.js

```diff
@@ -32,29 +43,36 @@ describe('#addPoolTransaction', () => {
     TransactionPool._storage.clear()
   })
 
-  test('Adds a transaction to the pool', () => {
+  test('Adds a transaction to the pool', async () => {
     TransactionPool.addPoolTransaction(tx1, bjj1)
+    await sleep(timeoutStore)
```

Comment:
Why do you need a timeout?

Reply:
Oh, you're right!! How about we return the promise then? I think it's a mistake on my part to not return it

Reply:
Returning a promise sounds great !! xD